charts/telepresence/values.yaml

```diff
@@ -0,0 +1,45 @@
+# The Traffic Manager only support running with one replica at the moment.
+# Configuring the replicaCount will be added in future versions of Telepresence
+# replicaCount: 1
+
+image:
+  repository: docker.io/datawire/tel2
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

Comment:
This isn't so much a problem with the chart, but if a user has a different version of the cli, it will try to update the image to the version of their cli. I think we have an issue to improve behavior around this, but just might be worth something to tell people who are using the chart, that it's important that users have the correct version of the CLI + have their `TELEPRESENCE_REGISTRY=image.repository` if they change that value.

Reply:
I think the answer is to document this for now, and very soon we'd like to make the go code use the helm chart for deploying the traffic-manager or have the go code generate the helm chart (but that's work for our team, and probably me) so for now I'd just make sure we have that documented wherever we add docs about the helm chart.